Index: src/main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main.py	(revision c21b3446d1703d6cfdfb4b2ddca5d4bef63814db)
+++ src/main.py	(date 1607023241964)
@@ -1,0 +1,4 @@
+import tkinter
+
+if __name__ == '__main__':
+    print(tkinter.TkVersion)
\ No newline at end of file
